README.md

```diff
@@ -1,2 +1,49 @@
-# space-tech-challenge-2022
-Space Tech Expo Challange 2022
+<p align="right">
+  <img width="25%" height="25%"src="images/klepsydra_logo.jpg">
+</p>
+
+# C++ Code challenge
+
+## What is missing the in the following code?
+Provide a solution that adds the minimal number lines to the code.
```

Comment:
Add this text at the start : A valid Sudoko solution is a 9 × 9 grid with digits such that each column, each row, and each of the nine 3 × 3 subgrids that compose the grid (also called "boxes", "blocks", or "regions") contains all of the digits from 1 to 9. The following code is intended to validate if a solution to Sudoku puzzle is correct or not.

Reply:
Done